x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.tsx

```diff
@@ -40,6 +41,7 @@ export const EmptyEngineOverview: React.FC = () => {
           </EuiButton>
         </EuiPageHeaderSection>
       </EuiPageHeader>
+      <FlashMessages />
```

Comment:
I need this for displaying the `setQueuedSuccessMessage` from [`CreateEngineLogic.actions.onCreateEngineSuccess`](https://github.com/elastic/kibana/pull/89156/files#diff-26e1600bafa52682720be1f18c7e96dc7b5b271c0baf49a0cb14e605e4c3b92bR71-R78)